### ISSUE TITLE
Support meilisearch v1.1.x - Multi-index search

### ISSUE THREAD

**Issue by BlitzBanana** [nutshell-lab/meilisearch-ex#2]
See: https://github.com/meilisearch/meilisearch/releases/tag/v1.1.0-rc.0

**Comment by fredwu** [comment]
Hi, I'm looking into using Meilisearch in my elixir project. Is this being actively worked on?

cc @ brunoocasali since you've previously offered help on https://github.com/robottokauf3/meilisearch-elixir/issues/10

Thanks folks! 🙏

**Comment by BlitzBanana** [comment]
@fredwu @brunoocasali

Hi,

First of all, thanks for your interest in this lib.

I'm using Meilisearch in production (0.25, then 1.0.x).
So, I started by forking https://github.com/robottokauf3/meilisearch-elixir to support 0.25.x and made a pull request.
It was eventually merged a year later, but in the meantime, I ended up creating my own lib https://github.com/nutshell-lab/meilisearch-ex for 1.0.0.

I'm using this in production, so, I would like to maintain it actively, but I lack the time to do it as soon as Meilisearch updates comes out.
That said, **I will gladly accept pull requests** and I will try to find the time to update to 1.2 new features.

**Comment by fredwu** [comment]
@BlitzBanana Thanks mate!

I've started looking into implementing some of the new APIs, starting from multi-index (got it working). I've also noticed a few small things in the code base that could use some improvements.

I'll issue a PR when it's ready. :)

**Comment by brunoocasali** [comment]
Hi @BlitzBanana and @fredwu!

That’s amazing to see the Elixir community taking care of that. Indeed I reached out robottokauf3 offering my help to make the work move forward on its side, but since I did not receive any official “reply” from him, I thought he didn’t have time to maintain it, and that’s perfectly fine.

Out of the blue, I’m not an Elixir expert, but I’m learning it in my spare time. And of course, I would love to have an Elixir integration, but I don’t have the time and knowledge to maintain one entirely.

If you agree on @BlitzBanana, I would love to make this repository like the other meilisearch integrations. Then in the future, we could move it to our Meilisearch organization if you agree, which could bring more awareness and improve the adoption of the library.
From the Meilisearch side, I could also map which feature is present or not regarding our releases (we have some automation for that).
